include/cuda/std/detail/libcxx/include/__functional/invoke.h

```diff
@@ -394,6 +400,7 @@ __invoke(_Fp&& __f, _A0&& __a0)
 
 // bullet 7
 
+_LIBCUDACXX_DISABLE_EXEC_CHECK
```

Comment:
We should be aware, that this will disable execution checks for most of the library. 

I am all in favor for that, but it will lead to bugs

Reply:
I was wondering about the potential for this problem. But I'm not sure what can be done about it... 🤔

Reply:
This has been true in Thrust for time immemorial. It's nearly impossible to have generic C++ code without heavy use of this pragma.